cmd/interval/list/list.go

```diff
@@ -46,23 +48,8 @@ func listHandler(cmd *cobra.Command, args []string) (err error) {
 		return err
 	}
 
-	pw := viper.Get("writer").(io.WriteCloser)
-	w := new(tabwriter.Writer)
-	w.Init(pw, 0, 8, 1, '\t', 0)
-	fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%s\t%s\t%s\t\n", "Interval ID", "Name", "Start",
-		"End", "Frequency", "Cron", "RunOnce")
-	for _, interval := range intervals {
-		fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%s\t%s\t%t\t\n",
-			interval.ID,
-			interval.Name,
-			interval.Start,
-			interval.End,
-			interval.Frequency,
-			interval.Cron,
-			interval.RunOnce,
-		)
-	}
-	w.Flush()
+	formatter := formatters.NewFormatter("commandList", intervalTemplete, nil)
```

Comment:
"intervalList" instead of commandList ?

Reply:
could we just fix it to some default given we have only one instance at any time in cli/

Reply:
Done - one default name is used